style.css

```diff
@@ -61,3 +73,176 @@ h3 {
   background-color: aqua;
   transition: 1s;
 }
+
+.card {
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 5rem;
+  width: 90vw;
+  outline: 1px solid rgb(208, 205, 205);
+  border-radius: 1rem;
+  background-color: rgb(255, 255, 255);
+}
+
+.cardsTitle {
+  font-size: 2.3rem;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  margin-left: 1.4rem;
+  margin-bottom: -0.8rem;
+}
+
+.subTitleList {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-left: -1rem;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+.techList {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-left: -1rem;
+  gap: 0.5rem;
+}
+
+.pictures {
+  width: 90%;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 1rem;
+  margin-bottom: -1rem;
+}
+
+.pictureContainer {
+  display: flex;
+  align-items: center;
+}
+
+.canopy {
+  color: #344563;
+  font-size: 0.9rem;
+  line-height: 1rem;
+  font-weight: 600;
+}
+
+.dots {
+  height: 0.6rem;
+}
+
+.grayWords {
+  font-weight: 600;
+  font-size: 0.9rem;
+  line-height: 1rem;
+  color: #7a869a;
+}
+
+.pInfo {
+  font-size: 0.95rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  margin-left: 1.5rem;
+}
+
+.techs {
+  background-color: #ebebff;
+  font-size: 0.9rem;
+  color: #6070ff;
+  padding-top: 0.3rem;
+  padding-bottom: 0.3rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 8px;
+  font-weight: 500;
+}
+
+.seeProjectButton {
```

Comment:
- [ ] Great job adding the hover and enabled buttons interactions, however, please kindly add the active (when it's pressed) interaction as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;`

### EXAMPLE:
**for pressed state**
```
.seeProjectButton:active {
   background-color: #2230D2;
}
```

Reply:
OK! Done.
![image](https://github.com/DamianDib9/personal-portfolio/assets/121461459/c350ec01-f113-4d39-87e7-9771fa8826a1)